1st try

PaulFlesh opened the pull request.

PaulFlesh committed:
1st try

andryxxa93 commented on src/components/BurgerIngredients/BurgerIngredients.tsx:
```diff
@@ -45,27 +38,27 @@ export default function BurgerIngredients() {
+  }, [dispatch, items]); // eslint-disable-line
 
-  const ingredientsScrollRef = useRef(null);
+  const ingredientsScrollRef: any = useRef(null); // any
```
**Можно лучше**
Тут можно избавиться от `any` так - `  const ingredientsScrollRef: RefObject<HTMLDivElement> = useRef(null);
`
`ingredientsScrollRef!.current!.scrollTop = scrollTop;`

andryxxa93 reviewed (approved):
Супер, вы отлично поработали!
Что понравилось:
- Вы не использовали ts-ignore
- Использовали any только в крайних случаях
- Хорошая типизация хранилища

Поздравляю! Работа принята.
Желаю удачного продолжения обучения!

PaulFlesh committed:
Мелкие исправления

PaulFlesh committed:
Давно пора

Исправлена работа корзины, удалены лишние экшены, починен профиль при обновлении страницы, добавлена стабильная сортировка заказов в кабинете
